.travis.yml

```diff
@@ -0,0 +1,26 @@
+os:
+  - linux
+  - osx
+
+language: go
+sudo: false
+
+go:
+  - 1.7
+
+before_install:
+  - go get github.com/mattn/goveralls
+  - go get golang.org/x/tools/cmd/cover
+  - go get github.com/whyrusleeping/gx
+  - go get github.com/whyrusleeping/gx-go
```

Comment:
Eeeh I'm sorry the way it was, was absolutely correct. This just installs the gx/test tooling, not actual code dependencies (which go get -d is for).

Could you change it back? Sorry :D